capellambse/model/crosslayer/cs.py

```diff
@@ -102,40 +104,39 @@ class PhysicalLink(PhysicalPort):
     linkEnds = c.AttrProxyAccessor(
         PhysicalPort, "linkEnds", aslist=c.ElementList
     )
-
-    physical_paths: c.Accessor
     exchanges = c.ProxyAccessor(
-        ComponentExchange,
-        xtypes="org.polarsys.capella.core.data.fa:ComponentExchangeAllocation",
+        fa.ComponentExchange,
+        xtypes=fa.XT_COMP_EX_ALLOC,
         aslist=c.ElementList,
         follow="targetElement",
     )
 
+    physical_paths: c.Accessor
+
 
-@c.xtype_handler("org.polarsys.capella.core.data.cs:PhysicalPath")
+@c.xtype_handler(XT_PHYS_PATH)
```

Comment:
This must be `None`; the first argument to `xtype_handler` is the architectural layer that this object can appear on (where `None` means it can appear anywhere, suitable for cross-layer objects).